rasr/feature_scorer.py

```diff
@@ -126,25 +127,53 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
```

Comment:
Please add the types in the function head so the IDE can use them.

Reply:
what type you mean?

Reply:
E.g. `label_log_posterior_scale: float = 1.0`

Reply:
instead of doing typing in the docstring

Reply:
I prefer doing docstring so that some explanation is provided

Reply:
Leave the docstring itself I am only talking about  the type hint.

Reply:
I suggest you open a new PR where you do this consistently for all classes in this module, because it does not make sense to do this for one and not for all others

Reply:
You mean for all classes in this module?

Reply:
Mainly for the new/updated code. Sometimes people then directly change it for the whole file, but this is up to you.

Reply:
> It was decided

I do not recall this decision. What is exactly the decision? So I need to now add documentation for all classes and add typings for all of them in `feature_scorer.py`?

Reply:
> Mainly for the new/updated code.

IMO, this does not make sense here, we have similar classes and they should all follow similar treatment. I can add the typing to this, but I suggest you open another PR for other classes

Reply:
No. I am just asking about moving the typing from the docstring to the function header of the updated class as it is done in every newer PR updating code in this repo.

Reply:
ok, will do

Reply:
I dont think its that big of a problem to have both variants in one file, we can update them as we go along as we usually do.

Reply:
Benedikt is correct about this. Also, we should not use `str` for files but only `tk.Path`.